models/Profile.js

```diff
@@ -0,0 +1,62 @@
+import pool from '../lib/utils/pool.js';
+import { getQuotes } from '../utils/futurama-api.js';
+
+export default class Profile {
+  id;
+  name;
+  favoriteCharacter;
+  tagline;
+
+  constructor(row){
+    this.id = row.id;
+    this.name = row.name;
+    this.favoriteCharacter = row.favorite_character;
+    this.tagline = row.tagline;
+  }
+
+  static async insertProfile ({ name, favoriteCharacter, tagline }){
```

Comment:
just a thought but since you had the function and the favorite character, you could do something similar to the update so that way the user wouldn't need to create a tagline at the beginning. Just their name and favoriteCharacter. The user experience might be a bit easier. 
This works too, just a thought!